### PR TITLE
Sofia/getting started

### PR DESCRIPTION
**Pull request by sofiiako**

### CODE REVIEW

File: Introduction/Getting started/Installation/task.md [hunk 1]
```diff
@@ -9,6 +9,27 @@ Let’s start your Rust journey! There’s a lot to learn, but every journey sta
...
+#### Windows
+1. Download rustup installer from https://win.rustup.rs/x86_64.
+2. Run it and follow the onscreen instructions.
+3. Restart your IDE.
+4. Setup Rust toolchain (press "Setup toolchain" in the pop-up "No Rust toolchain configured").
+5. You're good to go!
+
 ### Verification of Rust installation
 
 You need to set up the environment in a system below are few things which you should check before the work.
```
Comment (Geravant) [inline Introduction/Getting started/Installation/task.md:35]: I believe the default case would be to install the rust, so I suggest changing the "check before the work" to "do before the work" and move this sentence to the very top (before the "Unix" header)

File: Introduction/Getting started/Installation/task.md [hunk 2]
```diff
@@ -9,6 +9,27 @@ Let’s start your Rust journey! There’s a lot to learn, but every journey sta
...
+5. You're good to go!
+
+#### Windows
+1. Download rustup installer from https://win.rustup.rs/x86_64.
+2. Run it and follow the onscreen instructions.
+3. Restart your IDE.
+4. Setup Rust toolchain (press "Setup toolchain" in the pop-up "No Rust toolchain configured").
+5. You're good to go!
+
 ### Verification of Rust installation
```
Comment (Geravant) [inline Introduction/Getting started/Installation/task.md:33]: This header now seems a bit out of place/ Maybe we can just drop it?

**Review by Geravant — CHANGES_REQUESTED** [review]

**Comment by sofiiako** [comment]
@Geravant Added the requested minor changes in Installation: [29175bb](https://github.com/jetbrains-academy/rustlings-course/pull/48/commits/29175bb72d67806b4f301300781f6c783e6e6414)

**Review by Geravant — APPROVED** [review]

**Review by Geravant — APPROVED** [review]

File: Understanding Ownership/Move Semantics/Mutable Argument/task.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ## Mutable Argument
 
-Make me compile without adding new lines –– just changing existing lines! (no lines with multiple semicolons necessary!)
+Make me compile without adding new lines — just changing existing lines! (no lines with multiple semicolons necessary!)
 
 <div class="hint">
   The difference between this one and the previous ones is that the first line of `fn fill_vec` that had `let mut vec = vec;` is no longer there.
```
Comment (stephen-hero) [inline Understanding Ownership/Move Semantics/Mutable Argument/task.md:6]: Unlike the previous cases, the first line of `fn fill_vec`, which had `let mut vec = vec;`, is now gone.

File: Introduction/Getting started/Installation/task.md [hunk 4]
```diff
@@ -9,9 +9,30 @@ Let’s start your Rust journey! There’s a lot to learn, but every journey sta
...
+1. In Terminal run the command to download Rust toolchain installer:
+```text
+$ curl https://sh.rustup.rs -sSf | sh
+```
+2. Now run this:
+```text
+$ source $HOME/.cargo/env
+```
+3. Restart your IDE.
+4. Setup Rust toolchain (press "Setup toolchain" in the pop-up "No Rust toolchain configured").
```
[the remark this answers is not in the document]
Comment (sofiiako) [reply Introduction/Getting started/Installation/task.md:26]: "Setup" is the way it's spelled in IDEA
toolchain

File: Understanding Ownership/Move Semantics/Mutable Argument/task.md [hunk 5]
```diff
@@ -1,6 +1,6 @@
 ## Mutable Argument
 
-Make me compile without adding new lines –– just changing existing lines! (no lines with multiple semicolons necessary!)
+Make me compile without adding new lines — just changing existing lines! (no lines with multiple semicolons necessary!)
```
[the remark this answers is not in the document]
Comment (sofiiako) [reply Understanding Ownership/Move Semantics/Mutable Argument/task.md:3]: "Make me compile" is the standard wording for these tasks in the original course, it's used everywhere... Not sure we should change it.

File: Introduction/Getting started/Installation/task.md [hunk 6]
```diff
@@ -9,9 +9,30 @@ Let’s start your Rust journey! There’s a lot to learn, but every journey sta
...
+5. You're good to go!
+
+#### Windows
+1. Download rustup installer from https://win.rustup.rs/x86_64.
+2. Run it and follow the onscreen instructions.
+3. Restart your IDE.
+4. Setup Rust toolchain (press "Setup toolchain" in the pop-up "No Rust toolchain configured").
```
[the remark this answers is not in the document]
Comment (sofiiako) [reply Introduction/Getting started/Installation/task.md:33]: same as line 26

File: Understanding Ownership/Move Semantics/Mutable Argument/task.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (stephen-hero) [reply Understanding Ownership/Move Semantics/Mutable Argument/task.md:3]: I see.. Thought a lot about it, but without a context it's difficult to see all the connotations.. "Make me" sounds kinda weird, though, like - "Заставьте меня!" :)

File: Introduction/Getting started/Installation/task.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (stephen-hero) [reply Introduction/Getting started/Installation/task.md:26]: Yes, these two are often confused, even by interface writers, I guess. However, 'setup' is a noun, while 'set up' is a verb. What I would suggest here then is a compromise: Set up Rust toolchain (press "Setup toolchain" in the pop-up "No Rust toolchain configured").

[the remark this answers is not in the document]
Comment (sofiiako) [reply Introduction/Getting started/Installation/task.md:26]: Agreed)

File: Understanding Ownership/Move Semantics/Mutable Argument/task.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (sofiiako) [reply Understanding Ownership/Move Semantics/Mutable Argument/task.md:3]: This is exactly the meaning I though was actually intended...))

[the remark this answers is not in the document]
Comment (stephen-hero) [reply Understanding Ownership/Move Semantics/Mutable Argument/task.md:3]: Somewhat masochistic, but - whatever ))

**Pull request merged**